configuration/concepts/concepts_base.csv

```diff
@@ -0,0 +1,15 @@
+Uuid,Void/Retire,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,_version:1,_order:1000
+0aca2d79-1a16-42c8-b2c0-a4989bd55aec,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,Drug,N/A,,
+48728efa-b311-4696-a8d0-f2bc155ccd35,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,Drug,N/A,,
+b88c790f-f0db-4510-8e3c-9a7e012ef4b3,,TDF+3TC+ATZ/r ,,TDF+3TC+ATZ/r ,,TDF+3TC+ATZ/r ,,Drug,N/A,,
+1a6cdf69-1472-4b4c-9f57-e12db2681ee7,,TDF+3TC+LPVr ,,TDF+3TC+LPVr ,,TDF+3TC+LPVr ,,Drug,N/A,,
+a8f47b0b-1dde-4adf-8707-879b26159de5,,AZT+DRV+RTV ,,AZT+DRV+RTV ,,AZT+DRV+RTV ,,Drug,N/A,,
+a6aeea7f-620b-424e-a68e-38b28f4ed588,,ABC+DRV+RTV,,ABC+DRV+RTV,,ABC+DRV+RTV,,Drug,N/A,,
+66e89a39-4aa8-4d8b-b7b2-704fc678bc25,,AZT/3TC+EFV,,AZT/3TC+EFV,,AZT/3TC+EFV,,Drug,N/A,,
```

Comment:
@IipperMangesh This drug exists already in the current dev box and it has a concept associated with it.
The drugs from line 9 to 15 also exist already